users/models.py

```diff
@@ -236,8 +236,11 @@ class Guardian(GDPRModel, UUIDPrimaryKeyModel, TimestampedModel, SerializableMix
         related_name="guardians",
         blank=True,
     )
+    # NOTE: All guardians are initialized to have has_accepted_communication=True
+    # in users/migrations/0013_replace_has_accepted_marketing_with_communication.py,
+    # before they has implicitly accepted it when they have created an account.
```

Comment:
Is this information useful to have here in the future too? For me this feels more like a comment for a git commit message or someplace else rather than something needed here all the time IMHO. There's also typo "they has" -> "they have". I'm not saying you have to remove this text, but please would you think about this and if you also feel it would be better to have this information somewhere else, then you can of course move it, but I leave this consideration up to you, so do as you wish.